store/siren-action-behavior.html

```diff
@@ -30,7 +33,7 @@
 						return;
 					}
 					// if the field is specified multiple times, assume it is intentional
-					fields.append(field.name, field.value);
+					fields.push({name: field.name, value: field.value});
```

Comment:
```javascript
fields = action.fields.filter(function(field) {
    return field.value !== undefined;
});
```

Reply:
Actually, you could use `concat` to avoid using `forEach` + `push`

Reply:
no worries about the undefined value ones getting in that way?

Reply:
They will be a problem when converting to a `FormData` or `URLSearchParams`.

Reply:
ok, so I think using concat would be a problem then since they would get added as well

Reply:
You could filter the undefined ones first. I don't think it makes a huge difference. Might save a couple of lines to use filter/concat. But I'm okay with this.